### PR TITLE
Update versions in libraryDependencies

### PR DESCRIPTION
**Pull request by Philippus**

### CODE REVIEW

File: getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We assume you know [how to build a project in IntelliJ](building-a-scala-project
 * Add the ScalaTest dependency to your build.sbt file:
 
 ```
-libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.1" % "test"
+libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.5" % "test"
```
Comment (SethTisue) [inline getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.md]: is there a 3.0.5? there's a 3.0.4 and a 3.0.5-M1, but I don't think there's a 3.0.5

Comment (Philippus) [reply getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.md]: it's on maven central, released on 29-Jan-2018. Maybe it's not an official release?

Comment (SethTisue) [reply getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.md]: oh, okay: https://github.com/scalatest/scalatest-website/issues/102

**Comment by SethTisue** [comment]
thank you!

**Pull request merged**